data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/ScalarDbMode.java

```diff
@@ -1,20 +1,31 @@
-package com.scalar.db.dataloader.core;
+package com.scalar.db.dataloader.cli;
```

Comment:
Target branch has to be updated to master once the depending PR has been updated. Keeeping this conversation to prevent merging.